guidmap/getuidmap_test.go

```diff
@@ -1,19 +1,33 @@
-package kwok
+package guid
```

Comment:
We should avoid `guid` as a package name. It could cause some confusion since [UUIDs/GUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier) are commonly used.